File: Resolver/arc.py
class Arc:
    def __init__(self, city1, city2, distance):
        self.__city1 = city1
        self.__city2 = city2
        self.__distance = distance
    
    @property
    def city1(self):
        return self.__city1
        
    @property
    def city2(self):
        return self.__city2
    
    @property
    def distance(self):
        return self.__distance
        
    def ToString(self):
        return ", ". join((self.city1, self.city2, str(self.distance)))

File: Resolver/SolverBindingClasses/SolverBindingClasses/Arc.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SolverBindingClasses
{
    public class Arc
    {
        public Arc(int City1, int City2, double Distance)
        {
            this.City1 = City1;
            this.City2 = City2;
            this.Distance = Distance;
        }

        public int City1 { get; set; }
        public int City2 { get; set; }
        public double Distance { get; set; }
    }
}
